air/src/options.rs

```diff
@@ -122,13 +127,15 @@ impl ProofOptions {
     /// - `grinding_factor` is greater than 32.
     /// - `fri_folding_factor` is not 2, 4, 8, or 16.
     /// - `fri_remainder_max_degree` is greater than 255 or is not a power of two minus 1.
+    #[allow(clippy::too_many_arguments)]
     pub const fn new(
         num_queries: usize,
         blowup_factor: usize,
         grinding_factor: u32,
         field_extension: FieldExtension,
         fri_folding_factor: usize,
         fri_remainder_max_degree: usize,
+        batching_constraints: BatchingMethod,
         batching_deep: BatchingMethod,
     ) -> ProofOptions {
```

Comment:
Not for this PR, but we should refactor how we construct `ProofOptions` - maybe it should be more like a builder pattern.

Reply:
Created an issue for it